seqr/utils/search/hail_search_utils.py

```diff
@@ -203,44 +203,6 @@ def _get_sort_metadata(sort, samples):
     return sort_metadata
 
 
-def _parse_location_search(search):
```

Comment:
since parsing intervals from location search is now shared between hail search and clickhouse search, I moved the logic into the shared search utils file